modules/core/src/index.js

```diff
@@ -27,3 +27,4 @@ export {addColor} from './utils/color';
 export {leftPad, rightPad} from './utils/formatters';
 export {autobind} from './utils/autobind';
 export {default as LocalStorage} from './utils/local-storage';
+export {default as hiResTimestamp} from './utils/hi-res-timestamp';
```

Comment:
Nit: We usually follow the "verb-noun" convention for function/method names. If we wanted to follow that convention we could either just avoid renaming the method (it was already intended to be a high-res method), or call it `getHiResTimestamp()`.